packages/api/tests/endpoints/test-granules.js

```diff
@@ -1481,3 +1484,29 @@ test.serial('PUT with action move returns failure if more than one granule file
 
   filesExistingStub.restore();
 });
+
+test.serial('POST creates new granule in dynamoDB and postgres', async (t) => {
```

Comment:
I think we also should have two basic tests to show that that the granule doesn't appear in Dynamo if the PG write fails and vice versa

Reply:
The mocking in the collections endpoint and the unit tests might be a helpful guide:

https://github.com/nasa/cumulus/blob/master/packages/api/endpoints/collections.js#L110
https://github.com/nasa/cumulus/blob/master/packages/api/tests/endpoints/collections/create-collection.js#L375-L429

Reply:
Outside of the unit tests that already exist for that?

Reply:
I suppose given that the endpoint just calls `writeGranuleFromApi`, which does test this logic, we don't need to. It is possible that our endpoint logic could somehow change and we would want to be sure that the endpoint still behaves as expected, but YAGNI for now

Reply:
There's a bit more to the endpoint than there was before now that we lifted the finding of collectionCumulusId out of the original.  I added other tests. But the endpoint itself, only uses PG granule models, and all of the logic for writing DynamoDB vs PG is encapsulated in writeGranuleFromApi so I think those cases are handled at the moment.  I can take out the test context because it's not being used in testing.